platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMapOptions.java

```diff
@@ -644,7 +644,24 @@ public MapboxMapOptions crossSourceCollisions(boolean crossSourceCollisions) {
    */
   @NonNull
   public MapboxMapOptions localIdeographFontFamily(String fontFamily) {
-    this.localIdeographFontFamily = fontFamily;
+    this.localIdeographFontFamily = FontUtils.extractValidFont(fontFamily);
+    return this;
+  }
+
+  /**
+   * Set a font family from range of font families for generating glyphs locally for ideographs in the
```

Comment:
It'd be worth mentioning that it's a fallback system - that the first font from the list that's valid is going to get picked, if no matches, it defaults to sans-serif, unless `null`.